README.md

```diff
@@ -200,6 +200,60 @@ published on Hackage, or part of Stackage snapshots, using Bazel.
 
 [hazel]: https://github.com/formationai/hazel
 
+## Troubleshooting
+
+* If you see error messages complaining about missing `as` executable:
+
+  ```
+  cc: error trying to exec 'as': execvp: No such file or directory
+  `cc' failed in phase `Assembler'. (Exit code: 1)
+  ```
+
+  It means that your `gcc` cannot find `as` by itself. This happens only on
+  certain operating systems such as Ubuntu. We need to make `as` visible
```

Comment:
We should be more precise. Recent ubuntu's work fine. If anything it's Fedora that's more of a problem. In any case, what she should say is, this happens on certain platforms where GCC was built without the `--with-as` and `--with-ld` configure flags.